colossalai/inference/engine/policies/llama.py

```diff
@@ -20,20 +20,17 @@
 from ..modeling.llama import LlamaInferenceForwards
 
 try:
-    from colossalai.kernel.triton import rmsnorm_forward
-
+    from lightllm.models.llama.triton_kernel.rmsnorm import rmsnorm_forward as lightllm_rmsnorm_forward
     HAS_TRITON_RMSNORM = True
 except:
-    print("you should install triton from https://github.com/openai/triton")
+    print("you should install triton from GitHub - openai/triton: Development repository for the Triton language and compiler")
     HAS_TRITON_RMSNORM = False
```

Comment:
This looks really weird to me, if I did not install lightllm, the message is to ask me to install triton?